script/seed.js

```diff
@@ -1,58 +1,56 @@
 
-const {db, models: {User} } = require('../server/db')
+const products = [{
+  name: 'Blue Hickory',
+  description: 'Here’s our second time working with hickory! This gorgeous bowl has been an eye catcher from the moment we poured it, it’s beauty is just out of this world!',
+  material: 'Hickory',
+  epoxyColor: 'blue',
+  price: 200.00,
+  category: 'Bowl',
+  image: 'https://scontent-mia3-1.xx.fbcdn.net/v/t39.30808-6/276123626_158077886599088_4538128186652999978_n.jpg?_nc_cat=104&ccb=1-5&_nc_sid=8bfeb9&_nc_ohc=N0tLt3VXmlEAX9KWbmJ&_nc_ht=scontent-mia3-1.xx&oh=00_AT9sX3xrEBJ8b8LcArmdNUkEs3z7vHDEfJFcAb386oHKuA&oe=6264C172',
+  customizable: true
+}, {
+  name: 'Jewelry box',
+  description: 'Get your own custom stained box, the perfect gift idea for any event your attending!Our first time working with hickory, and it definitely won’t be our last! We hope you love this small beautiful bowl, perfect for whatever you need, as much as we do!',
+  material: 'Poplar',
+  epoxyColor: 'wax',
+  price: 20.00,
+  category: 'Box',
+  image: 'https://scontent-mia3-2.xx.fbcdn.net/v/t39.30808-6/275627014_156467416760135_2314966038615759731_n.jpg?_nc_cat=107&ccb=1-5&_nc_sid=8bfeb9&_nc_ohc=X7_hcUuQyxQAX--erfZ&_nc_ht=scontent-mia3-2.xx&oh=00_AT9ge395gMEXPAoEfnTAJRtn62hwslJqeq46J2aaBikovA&oe=62645900',
+  customizable: true
+}, {
+  name: 'Hickory bowl',
+  description: 'Our first time working with hickory, and it definitely won’t be our last! We hope you love this small beautiful bowl, perfect for whatever you need, as much as we do!',
+  material: 'Hickory',
+  epoxyColor: 'wax',
+  price: 30.00,
+  category: 'Bowl',
+  image: 'https://scontent-mia3-2.xx.fbcdn.net/v/t39.30808-6/275585953_156078566799020_104830131098927512_n.jpg?_nc_cat=110&ccb=1-5&_nc_sid=8bfeb9&_nc_ohc=ttD-1p2WX-8AX_fIVbO&_nc_ht=scontent-mia3-2.xx&oh=00_AT9qXe438DDsY_0UUem_CRlsIt0ftE2Ef4j_nZ1ZrdqpkA&oe=62649B61',
+  customizable: true
+}]
 
-/**
- * seed - this function clears the database, updates tables to
- *      match the models, and populates the database.
- */
-async function seed() {
-  await db.sync({ force: true }) // clears db and matches models to tables
-  console.log('db synced!')
-
-  // Creating Users
-  const users = await Promise.all([
-    User.create({ username: 'cody', password: '123' }),
-    User.create({ username: 'murphy', password: '123' }),
-  ])
-
-  console.log(`seeded ${users.length} users`)
-  console.log(`seeded successfully`)
-  return {
-    users: {
-      cody: users[0],
-      murphy: users[1]
-    }
-  }
-}
-
-/*
- We've separated the `seed` function from the `runSeed` function.
- This way we can isolate the error handling and exit trapping.
- The `seed` function is concerned only with modifying the database.
-*/
-async function runSeed() {
-  console.log('seeding...')
+async function seed () {
   try {
-    await seed()
+    await db.sync({ force: true })
+
+    await Promise.all(products.map(product => {
```

Comment:
I am not sure if this will change anything - but maybe use a forEach here since we aren't using whatever is returned from map

Reply:
we followed seed files in other projects for this